.github/workflows/dotnet.yml

```
@@ -27,7 +30,7 @@ jobs:
         run: dotnet restore
 
       - name: 🏗️ Build solution
-        run: dotnet build --no-restore --configuration Release
+        run: dotnet build OPS.sln --no-restore --configuration Release
 
       - name: ✅ Run tests
-        run: dotnet test --no-build --configuration Release --verbosity normal
\ No newline at end of file
+        run: dotnet test OPS.sln --no-build --configuration Release --verbosity normal
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)